src/series_components/series-cell.js

```diff
@@ -12,13 +12,19 @@ var realm = require('../database/class');
 // add category and experience level to cell information
 module.exports = React.createClass({
   render: function() {
+    var styleWrapper = styles.seriesWrapper
+    var styleText = styles.seriesName
+    if (this.props.series.active === true) {
```

Comment:
Similar refactor, since this.props.series.active will return either true or false, you don't need the comparison. It will run or it won't run based on what the object query returns.